docs/src/pages/components/chatbot/warning.angular.mdx

```diff
@@ -0,0 +1,8 @@
+import { Alert } from '@aws-amplify/ui-react';
+
+<Alert variation="info" heading="Web Component">
+This component has not yet been converted to a native implementation.
+
+If you’re migrating from version 1 of the `@aws-amplify/ui-[framework]`, please see [Usage](#usage) below to see the updated import with `/legacy` path.
```

Comment:
I think we can specify the package name here and in other `warning.*.mdx`:

```suggestion
If you’re migrating from version 1 of the `@aws-amplify/ui-angular`, please see [Usage](#usage) below to see the updated import with `/legacy` path.
```

Reply:
Will do!